prime-angular-frontend/src/app/modules/enrolment/shared/services/enrolment.service.ts

```diff
@@ -69,10 +69,7 @@ export class EnrolmentService implements IEnrolmentService {
    * - Licences "Named in IM Reg" or "Licensed to Provide Care"
    */
   public canRequestRemoteAccess(certifications: CollegeCertification[], careSettings: CareSetting[]): boolean {
```

Comment:
Can you please update the code comments?

Reply:
the function comment is updated.